src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -160,6 +160,25 @@ else if(loans.getTotalRecords() == 1) {
     return result.combineAfter(userRepository::getUser, Loan::withUser);
   }
 
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findClosedLoans(int fetchLoansLimit) {
```

Comment:
Could methods _findClosedLoans_ and _findClosedLoansForUser_ be overloaded?

Reply:
Sure